client/js/components/procedure/admin/DpNewProcedure/DpNewProcedure.vue

```diff
@@ -7,53 +7,288 @@
   All rights reserved
 </license>
 
+<template>
+  <form
+    name="xsubmititem"
+    :action="Routing.generate('DemosPlan_procedure_new')"
+    enctype="multipart/form-data"
+    method="post"
+    data-dp-validate>
+    <input
+      type="hidden"
+      name="_token"
+      :value="token">
+    <input
+      type="hidden"
+      name="action"
+      value="new">
+    <input
+      type="hidden"
+      name="r_master"
+      value="false">
+    <!-- allow publishing of Statements by default -->
+    <input
+      type="hidden"
+      name="r_publicParticipationPublicationEnabled"
+      value="1">
+    <div v-if="hasPermission('feature_use_plis')">
+      <input
+        type="hidden"
+        name="r_name"
+        value="">
+      <input
+        type="hidden"
+        name="r_externalDesc"
+        value="">
+      <input
+        type="hidden"
+        name="r_mapExtent"
+        value="">
+    </div>
+    <fieldset>
+      <addon-wrapper hook-name="bimschg.request" />
+      <div v-if="hasPermission('feature_use_plis')">
+        <dp-form-row>
+          <dp-select
+            id="r_plisId"
+            :label="{ text: Translator.trans('name'), hint: Translator.trans('explanation.plis.procedurename') }"
+            name="r_plisId"
+            :options="plisNameOptions" />
+        </dp-form-row>
+        <dl class="u-nojs-hide--block">
+          <dt
+            v-text="Translator.trans('public.participation.desc')"
+            class="weight--bold" />
+          <dd
+            id="js__plisPlanungsanlass"
+            class="list-style-none" />
+        </dl>
+      </div>
+      <dp-form-row v-else>
+        <dp-input
+          data-cy="newProcedureTitle"
+          id="r_name"
+          :label="{ text: Translator.trans('name') }"
+          :maxlength="200"
+          name="r_name"
+          required
+          type="text" />
+      </dp-form-row>
+      <!--      {% block master %}-->
```

Comment:
nitpick
```suggestion
      <!-- block master -->
```

Reply:
I'd suggest to delete the line entirely, as it does not add any value anymore, once the project PRs are merged (and the block is removed).

Reply:
I removed it, because it has no purpose and I just forgot it. Done in [7c7706b](https://github.com/demos-europe/demosplan-core/pull/752/commits/7c7706b9bf34fcb618e07312e18830b6344b3921)